packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -212,7 +212,9 @@ function WorklistDataGrid(props) {
   const [focusedCol, setFocusedCol] = useState(0);
   const [ariaLiveMessage, setAriaLiveMessage] = useState(null);
   const [cellAriaLiveMessage, setCellAriaLiveMessage] = useState(null);
-  const rowSelection = useRef({ isRowSelectionModeToggledByGrid: false });
+  const isRowSelectionModeToggledByGrid = useRef(false);
+  const inShiftUpDownMode = useRef(false);
+  const multiSelectRange = useRef({ start: null, end: null });
```

Comment:
Maybe it would be easier to maintain the logic if we used two reference variables instead of an object with properties.

Reply:
I think that in this case, it is better to keep them together. A range should have a start and an end when one is established.